Pre-commit hook fails

### Expected behavior

`❯ yarn checking-typescript` should exit without errors.

### Actual behavior

```shell
❯ yarn checking-typescript                     
yarn run v1.17.3
$ node_modules/typescript/bin/tsc --noEmit
node_modules/@types/Strophe/index.d.ts:1111:3 - error TS2666: Exports and export assignments are not permitted in module augmentations.

1111   export = Strophe;
       ~~~~~~


Found 1 error.

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### Steps to reproduce the behavior
1. `❯ git clone git@github.com:jsxc/jsxc.git`
2. `❯ cd jsxc`
3. `❯ yarn`
4. `❯ yarn checking-typescript`

### Environment
- **JSXC version:** `4.0.0-rc.1`
- **Node version:** `10.15.3`
- **Yarn version:** `1.17.3`

--- a/package.json
+++ b/package.json
@@ -40,7 +40,7 @@
     "@types/mocha": "^5.2.7",
     "@types/node": "^12.6.8",
     "@types/sinon": "^7.0.13",
-    "@types/strophe": "^1.2.30",
+    "@types/strophe.js": "^1.2.0",
     "archiver": "^3.0.0",
     "chai": "^4.2.0",
     "clean-webpack-plugin": "^3.0.0",

--- a/src/connection/xmpp/ConnectHelper.ts
+++ b/src/connection/xmpp/ConnectHelper.ts
@@ -96,10 +96,6 @@ function prepareConnection(url: string): Strophe.Connection {
       };
    }
 
-   if (connection.caps) {
-      connection.caps.node = 'http://jsxc.org/';
-   }
-
    SM.changeState(SM.STATE.ESTABLISHING);
 
    return connection;

